### ISSUE TITLE
Scissors panel did not display as expected

### ISSUE THREAD

**Issue by hpb0412** [PeterPanen/storybook-addon-scissors#3]
Hi @PeterPanen ,
Here is what happened to my scissors panel
<img width="489" alt="screen shot 2017-10-17 at 3 10 32 pm" src="https://user-images.githubusercontent.com/7277539/31653660-8a5a0388-b34d-11e7-90c7-fa7f6a9ded06.png">

I am using storybook version 3.2.12

**Comment by jmca** [comment]
@hpb0412 Do you have a css-loader in your `webpack.config.js`?

**Comment by hpb0412** [comment]
@jmca: I used these for my css: style-loader, css-loader, postcss-loader
here it is:
```javascript
        test: /\.css$/,
        use: [
          {
            loader: 'style-loader',
          },
          {
            loader: 'css-loader',
            options: {
              modules: true,
              importLoaders: 1,
            },
          },
          {
            loader: 'postcss-loader',
            options: {
              config: {
                path: POSTCSS_CONFIG_FILE,
              },
            },
          },
        ],
```

**Comment by myrlogriz** [comment]
I have the same issue
"webpack": "^3.8.1",
"storybook-addon-scissors": "^3.0.1",
"style-loader": "^0.19.0",
"css-loader": "^0.28.4",

Webpack config also have style-loader and css-loader. The previous version of storybook-addon-scissors + kadira + webpack 1 worked perfectly together

Would be nice if issue would be solved, because that would be nice finally use the new storybook with this plugin

**Comment by myrlogriz** [comment]
@hpb0412 the funniest thing:
I removed  **modules: true** from options in css-loader and everything works now :)
So try it too, interesting to see how it will be for you


```
      {
        test: /\.css$/,
        use: [
          {
            loader: 'style-loader'
          },
          {
            loader: 'css-loader'
          }
        ]
      },
```

**Comment by PeterPanen** [comment]
@NataliaSheviakova Thanks for the info.
So it seems if you use storybook with a custom webpack config, and with modules enabled in css-loader it breaks?

I will try to look into this as soon as i have the time!

**Comment by myrlogriz** [comment]
@PeterPanen Yep, correct :) Thank you

**Comment by lucasconstantino** [comment]
Up

**Comment by lucasconstantino** [comment]
Up 2x

**Comment by PeterPanen** [comment]
Sorry for the delay.
I just published a new version 4.0.1, which should fix the problem.

Please report if you still have this issue after updating.
